tests/buffering.rs

```diff
@@ -317,6 +316,7 @@ fn test_reclaim_disk_space() {
         .map(|m| m.len())
         .sum();
 
-    println!("after {}, before {}", after_disk_size, before_disk_size);
-    assert!(after_disk_size < before_disk_size);
+    // Ensure that the disk after is _atleast_ more than half as small
+    // as it was before.
```

Comment:
"at least more than half as small" is breaking my brain

Reply:
haha, yeah I thought it made sense coming out of my brain but I guess hearing it out it doesn't. Let me fix that.